jcg_testcases/src/main/scala/TestCaseExtractor.scala

```diff
@@ -1,6 +1,8 @@
 import java.io.File
 
 trait TestCaseExtractor {
+    val pathSeparator: String = File.pathSeparator
```

Comment:
Can't these be in the TestCaseExtractor object? It is the companion object to this trait, isn't it?